release/release_gold.sh

```diff
@@ -98,6 +98,8 @@ function main {
 
 	lc_time_run test_boms
 
+	lc_time_run prepare_next_release_branch
```

Comment:
@kiwm, please make sure to execute this function just when releasing the quarterly releases